athena-federation-integ-test/README.md

```diff
@@ -0,0 +1,159 @@
+# Integration-Test Module
+
+The Integration-Test module provides end-to-end testing capabilities, and is available
+to all lambda connectors developed using the Athena Federation SDK.
+
+## How It Works
+
+In order to test the connectors end-to-end, several infrastructure resources need to be
+provisioned and deployed (e.g. DB instance, Lambda function, etc...) This module accomplishes
+that by allowing AWS CloudFormation to manage the infrastructure resources. All an
+integration-test writer needs to do is provide an implementation for a handful of functions,
+and the Integration-Test module will do the rest.
+
+This module provides the following benefits:
+* Automatically provisions all infrastructure resources prior to testing, and de-provisions
+them immediately after.
+* Provides a set of public APIs that can be used to send queries via Athena using the lambda
+connector.
+
+## Writing Integration Tests
+
+This section explains the steps necessary to create integration tests using the
+Integration-Test module. For an actual code example, see the DynamoDB connector
+[DynamoDBIT integration-test class](https://github.com/awslabs/aws-athena-query-federation/blob/master/athena-dynamodb/src/test/java/com/amazonaws/athena/connectors/dynamodb/DynamoDBIT.java).
+
+### Dependencies
+
+Add the Integration-Test module as a test dependency in the specific connector's `pom.xml` file:
+
+```xml
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>athena-federation-integ-test</artifactId>
+            <version>1.0</version>
+            <scope>test</scope>
+        </dependency>
+```
+
+### Naming Convention
+
+All integration-test classes need to adhere to the naming convention `<class>IT`
+(e.g. `public class DynamoDBIT`). The same goes for the integration-tests themselves.
+
+### Writing Integration Tests
+
+Import the `@Test` annotation from `org.testng.annotations` and extend the Integration-Test
+class (`IntegrationTestBase`):
+
+```java
+import org.testng.annotations.Test;
+
+public class MyConnectorIT extends IntegrationTestBase
```

Comment:
Naming convention should suggest ItegTest instead of IT

Reply:
Changed to `MyConnectorIntegrationTest`.